snuba/utils/streams/backends/kafka.py

```diff
@@ -637,6 +637,31 @@ def build_kafka_consumer_configuration(
     }
 
 
+def build_kafka_configuration(bootstrap_servers: Sequence[str],) -> Mapping[str, Any]:
+    return {
+        "bootstrap.servers": ",".join(bootstrap_servers),
```

Comment:
according to [this](https://docs.confluent.io/current/clients/confluent-kafka-python/#pythonclient-configuration), 
> The supported configuration values are dictated by the underlying librdkafka C library
which [supports](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)

```python
        ...
        "security.protocl": "sasl_ssl",
    }
```

Reply:
Hi @olliezhu, thank you for your review.

Please note this commit was an intermediary step in the direction of refactoring the code to support more meaningful configuration. 

The end goal is allowing the librdkafka supported configuration values to be set in the `settings.py`. 

I hope this answers your comment.